src/items.rs

```diff
@@ -1,6 +1,15 @@
 //! Some item definitions used in instances to provide a virtual representative
 //! structure of `.cnf` or `.sat` files and their associated clauses or formula.
 
+use std::fmt::Display;
+
+#[cfg(windows)]
+const LINE_ENDING: &str = "\r\n";
+
+#[cfg(not(windows))]
+const LINE_ENDING: &str = "\n";
```

Comment:
According to the docs here you don't need this distinction in Rust:
https://doc.rust-lang.org/stable/src/std/macros.rs.html#65